packages/edit-site/src/components/template-switcher/theme-preview.js

```diff
@@ -31,7 +31,7 @@ function ThemePreview( {
 				alt={ 'Theme Preview' }
 			/>
 			<div className="edit-site-template-switcher__theme-preview-description">
-				{ truncate( description, {
+				{ truncate( description.raw, {
```

Comment:
Why isn't this the rendered version? The rendered version has `wptexturize` applied which handles things like curly quotes.

Reply:
My thinking was that the rendered version might contain HTML markup (stuff like `<a />` or `<em />`) and would thus require us to `__dangerouslySetInnerHtml` to render it in React, which I wanted to avoid.

Reply:
It could, but it has `wp_kses` applied to be safe.

Reply:
Hmm, okay. I can see that for `name`, but with `description` here, there's an extra complication -- I'm `truncate()`ing, so we might end up with an opening HTML tag that messes up the rest of the markup :confused:

Reply:
ec6f16d6da

Reply:
It looks like this could be possible with https://github.com/arendjr/text-clipper.

Reply:
Thanks, I wasn't aware of that! I think it's worth considering, but since we'd require an extra dependency for truncating the theme description down to ~120 chars, it might be controversial, so I'd rather not block this PR on it. We can open a separate issue/PR tho!